.github/workflows/master-a380x.yml

```diff
@@ -0,0 +1,95 @@
+# Build script for the A380X on master branch
+# Build independent of the A32NX to avoid side effects if one or the other fails
+
+# TODO: Add a check if something changed outside of the fbw-a32nx folder (which isn't of interest for this build)
+#       Consider that master pre release should always have a A32NX and A380x though
+
+name: master
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    # Prevent running this on forks
+    if: github.repository_owner == 'flybywiresim'
+    runs-on: ubuntu-latest
+    env:
+      A32NX_PRODUCTION_BUILD: 1
+      A32NX_INSTRUMENTS_BUILD_WORKERS: 2
+      MASTER_PRE_RELEASE_ID: 66067756
+      MASTER_PRE_RELEASE_TAG: assets/master
+      MASTER_ZIP_NAME: A380X-master.7z
+      VMASTER_PRE_RELEASE_ID: 32243965
+      VMASTER_PRE_RELEASE_TAG: vmaster
+      BUILD_DIR_NAME: master
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+      - name: Set BUILT_DATE_TIME
+        run: echo "BUILT_DATE_TIME=$(date -u -Iseconds)" >> $GITHUB_ENV
+      - name: Create .env file
+        run: |
+          echo A32NX_PRODUCTION_BUILD=1 >> fbw-a380x/.env
+          echo CLIENT_ID=\"${{ secrets.NAVIGRAPH_CLIENT_ID }}\" >> fbw-a380x/.env
+          echo CLIENT_SECRET=\"${{ secrets.NAVIGRAPH_CLIENT_SECRET }}\" >> fbw-a380x/.env
+          echo CHARTFOX_SECRET=\"${{ secrets.CHARTFOX_SECRET }}\" >> fbw-a380x/.env
+          echo SENTRY_DSN=\"${{ secrets.SENTRY_DSN }}\" >> fbw-a380x/.env
+          echo BUILD_DIR_NAME=${{ env.BUILD_DIR_NAME }} >> .env
+          echo LOCALAZY_READ_KEY=${{ secrets.LOCALAZY_READ_KEY }} >> .env
+          echo VITE_BUILD=false >> .env
+      - name: Build A380X
+        run: |
+          ./scripts/dev-env/run.sh ./scripts/setup.sh --clean
+          ./scripts/dev-env/run.sh ./scripts/build_a380x.sh --no-tty -j 4
+          rm -rf fbw-a380x/src
+          docker system prune -af
+      - name: Build ZIP files
+        run: |
+          df -h
+          node ./scripts/fragment_a380x.js
+          cp ./fbw-a380x/out/build-modules/modules.json ./fbw-a380x/out/flybywire-aircraft-a320-neo/install.json
```

Comment:
Is out/flybywire-aircraft-a320-neo right here?

Reply:
I reworked this into  one `master.yml` workflow.